scripts/build/cc/gcc.sh

```diff
@@ -447,13 +449,15 @@ do_gcc_core_backend() {
         core_LDFLAGS+=("-lm")
     else
         if [ "${CT_CC_GCC_STATIC_LIBSTDCXX}" = "y" ]; then
-            # this is from CodeSourcery arm-2010q1-202-arm-none-linux-gnueabi.src.tar.bz2
-            # build script
-            # INFO: if the host gcc is gcc-4.5 then presumably we could use -static-libstdc++,
-            #       see http://gcc.gnu.org/ml/gcc-patches/2009-06/msg01635.html
-            host_libstdcxx_flags+=("-static-libgcc")
-            host_libstdcxx_flags+=("-Wl,-Bstatic,-lstdc++,-Bdynamic")
-            host_libstdcxx_flags+=("-lm")
+            if [ "${CT_GCC_older_than_6}" = "yes" ]; then
```

Comment:
Should be `"${CT_GCC_older_than_6}" = "y"`

Reply:
I also wonder if this could be

`if [ "${CT_CC_GCC_STATIC_LIBSTDCXX}" = "y" -a "${CT_GCC_older_than_6}" = "y" ]`

which would make it easier to clean up when we drop the older GCCs